docs/installation.md

```diff
@@ -1,3 +1,65 @@
-# Installation
+# Installing Dusty
 
-Coming Soon
+## Requirements
+You must have the following installed in order to run Dusty:
+
+ * nginx
+ * Virtualbox
+ * boot2docker
+ * Docker Compose
+
+Not all versions of these programs will work with Dusty.  Dusty
+will warn you if you have a version installed that may be too old.
+We recommend brew-installing the latest versions of each of these
+programs.
+
+## Installation
+
+We provide a install script with each release.  This install script
+takes the following actions:
+
+* Downloads the Dusty binary, and places it at `/usr/local/bin/dusty`
+* Unloads any existing Dusty Daemon plist from `/System/Library/LaunchDaemons/org.gamechanger.dusty.plist`
+* Downloads our plist file to the same location
+* Runs dusty with `--preflight-only`, which will check some components of your system setup
+* Loads the plist that was downloaded, if the previous step was successful
+
+To download and run this install script, run
+```
+bash -c "`curl -L https://github.com/gamechanger/dusty/releases/download/0.1.1/install.sh`"
+```
+
+## Setup
+
+After you install Dusty, you should run `dusty setup` to do some
+necessary configuration.
+
+#### Username
+
+Dusty runs some commands as this user that you specify.  Usually the default
+here should be fine. Setting this to `root` is probably a **bad** idea.
+
+#### Specs Repo
+
+Dusty needs to know where your Dusty specs live.  You can enter a git
+repository here if you want to manage them that way (recommended).  You
+can also just specify a local file path.
```

Comment:
Should mention the example specs repo and link to the getting started section in case they want to run through that.